package.json

```diff
@@ -14,7 +14,7 @@
     "prebuild:dist": "rm -rf dist/*",
     "build:dist": "rollup -c && rollup -c --environment PRODUCTION",
     "build:watch": "npm run build:lib -- --watch",
-    "test": "mocha \"./src/**/*.test.js\" --require babel-core/register --timeout 5000",
```

Comment:
If we find a way to disable enzyme and jsdom during the tests yes, otherwise they throw errors if ran inside a browser...